packages/data-connect/src/index.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * Firebase Data Connect
+ *
+ * @packageDocumentation
+ */
```

Comment:
Does this comment need to be above the license? Guidelines is to start with the license at the top if at all possible.

Reply:
All the rest of the packages have it above. https://github.com/firebase/firebase-js-sdk/blob/main/packages/auth/index.ts#L4

Reply:
Well then!